src/go/cmd/update-collection-v3/migrations/disable-thanos/migrate_test.go

```diff
@@ -0,0 +1,34 @@
+package disablethanos
+
+import (
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDisableThanos(t *testing.T) {
+	_, testFileName, _, _ := runtime.Caller(0)
+	testDir := path.Dir(testFileName)
+	inputFileNames, err := filepath.Glob(path.Join(testDir, "testdata", "*.input.yaml"))
+	require.NoError(t, err)
+	for _, inputFileName := range inputFileNames {
+		t.Run(path.Base(inputFileName), func(t *testing.T) {
+			inputFileContents, err := ioutil.ReadFile(inputFileName)
+			require.NoError(t, err)
+
+			outputFileContents, err := Migrate(string(inputFileContents))
+			require.NoError(t, err)
+
+			expectedOutputFileName := strings.TrimSuffix(inputFileName, ".input.yaml") + ".output.yaml"
+			expectedOutputFileContents, err := ioutil.ReadFile(expectedOutputFileName)
+			require.NoError(t, err)
+			assert.Equal(t, string(expectedOutputFileContents), outputFileContents)
+		})
+	}
+}
```

Comment:
Is it copy of `main_test.go`?

Reply:
Yes, sadly it is. This duplication should be removed, but I wanted to push this change over the fence already... Can we leave it for another PR?

Reply:
tbh imho, If you would like to make some easy solution now and improve it in the future, please remove thanos `migrate_test.go` and refactor `main_test` to iterate over subdirectories. At least it will reduce an overhead

Reply:
Removed the duplication.